app/views/exam_templates/index.html.erb

```diff
@@ -1,4 +1,19 @@
+<% content_for :head do %>
+    <%= render partial: 'boot',
+               formats: [:js],
```

Comment:
boot is taken as html if we don't specify the formats, so I only deleted handlers line!